### DISCUSSION TITLE
Please help to add image on main page

### DISCUSSION THREAD

**Discussion by cryptobandteam** [thuliteio/doks#993] [Q&A] [answered]
Hello.

I am using doks theme and can't find a way to add image on main page. For blog this command works fine 

`![Test Image](/2.jpg)
`
But on main page this is not working :( Please help

**Comment by h-enk** [comment]
Does removing the slash work:

```md
![Test Image](2.jpg) 
```

Did you place `2.jpg` in `./content/en/`?

Do you have a repo you could share?

**Reply by cryptobandteam** [reply]
Hello.

I am getting this error 

Error: add site dependencies: load resources: loading templates: "/opt/build/repo/layouts/2.jpg:204:1": parse failed: template: 2.jpg:204: unterminated character constant

Repo is 

https://github.com/cryptobandteam/murfalgo-web3-v2

**Comment by h-enk** [comment]
Remove `murf_algo_trades.png` from `./layouts/` — it's not a template

When I e.g. add to `./content/en/_index.md`:

```md
![Alt text](murf_algo_trades.png)
```

It displays as expected:

![Snag_212e586d](https://user-images.githubusercontent.com/3902872/214643528-566b1b29-286c-49fd-ab47-807b9ab7cee2.png)

**Reply by cryptobandteam** [reply]
Thank you, sir.

That works 

_When I e.g. add to ./content/en/_index.md_

But how to add image to the section with "Smart trading
with algorithmic indicator"?

**Reply by cryptobandteam** [reply]
![Uploading image.png…]()

**Comment by h-enk** [comment, marked as answer]
You're using Doks v0.5 with `@hyas/images` v0.2.1. Unfortunately, this version doesn't come with a generic partial. The upcoming version will make it super easy to add images that way.

For now, you could just insert an  `<img>` element and put the image in the `static` folder:

```html
<img class="img-fluid" src="senja.jpg">
```

![Snag_21866f07](https://user-images.githubusercontent.com/3902872/214668012-0e8f1917-6f38-443a-8c84-4a555ae3acef.png)

Also, check out the source code of [ScreenshotOne](https://screenshotone.com/) to see how they did things

**Reply by cryptobandteam** [reply]
Thank you so much, sir! That works for me. 

I tried to make a research of ScreenshotOne repo with no luck :( 

Really appreciate your help
